apps/pulsechain/pulsechain.star

```diff
@@ -0,0 +1,138 @@
+"""
+Applet: PulseChain
+Summary: Price of PLS and PLSX
+Description: Display the price of PLS and PLSX. Choose between testnet and mainnet prices. After PulseChain mainnet launch, an update will be pushed to this app to display the correct mainnet price.
+Author: bretep
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("cache.star", "cache")
+load("schema.star", "schema")
+load("secret.star", "secret")
+
+POST_HEADERS = {
+    "Content-Type": "application/json",
+}
+
+PLS_ICON_SM = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAAkAAAAICAYAAAArzdW1AAAACXBIWXMAAAWJAAAF
+iQFtaJ36AAABC0lEQVQYlT3Ku07CUACA4b/0ioA9saE5MSTA4mQURyZ8CQcfwUkX
+Z+oLGB/AgcfQURdXXVwcLAmBg6GxEi5p0R4HE4dv+wytNbVLokWDPrvwr8oX0NJ1
+UnN8G7WqSwalFd48Y1isEayBAo8K5ajGnXX0sbmRyvalAqlI4xaMEpokQMK5kTCw
+9itLEeTOQxA7HaksIRVp/JfT2QyYcWztueNeYNcJ7M1LkLkifHWQqiSk4jBW8N7m
+2ewbbtszfzo7Xii3TC3KZnFfXdPzRya1uUFtwalRGNsic7tx7nT93Omysg6Gk9xp
+TnKHSeZeneV2hNYaDRffZkOvyic69a/1NHzSb+E0fhSfQmvNL6VjbO/VkT3+AAAA
+AElFTkSuQmCC
+""")
+PLSX_ICON_SM = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAAgAAAAJCAYAAAAPU20uAAAACXBIWXMAAAWJAAAF
+iQFtaJ36AAABLklEQVQYlT3KTSiDAQAG4Hd8tljL/qSV5KeWvoxYc6CViIOQ5mYO
+kiaHTbIkcpCi2VKUA2scNKJxctFSDg67TVlJS+SgNdKW2k/bvteJ69Mj2xocawEg
+e9HrX/1Bfw4ADNH1zUSHeln9lmnFU21jgJBLD3XGPZIwH7hNoKcgK26nu71uORZt
+kzX5clWSUBW8Q6OdlV9r9+AGG+5WnCRR5rs8+QyLptWSrCJza9cNZ3VSjyKdixqv
+s/sAIADAqbk98GjS5MPjWh9QKIpXpdkb744EACAJktDGXUegi/VR56HDPqP9c5BE
+W3DeCjpKQmbu46K/zxMzdB3/h4GlBUGRmo6BU7SEHBM/cjFOiNJur81KEsLzSM6c
+r86GlMkqnyaiPDs3N0cs799NYiKVBoBfQPqMHBohfUwAAAAASUVORK5CYII=
+""")
+
+def main(config):
+    PULSE_URL = secret.decrypt("AV6+xWcEwLL4pRoJwUrtaXE3CI9igLnERsQGHOaKT71Tx9ZK4KNzX7RnM44TUUH331nV7qC1HZyDQjWz5dRSOfhR+FygZJuuAA0U6lVtMJ0ec/LhP7g6njwRFMazQm5WKVzIEiiBZSu8j1cFt8p0nvnIX/YIzdkHD7oGmBeC4yJJS00Ff2mwdgk7thwyxzVASOxPhfqHx0wzF5+V9Hqt4TQKLNHrZraKaMvAlPNwYg==")
```

Comment:
Could these URLs be public? That would allow other community members to contribute to the app and build new apps on the same API. It'd also make reviewing a bit simpler for us at Tidbyt. =)

Reply:
No, they are likely to change, which is why I have them hidden now. Once they are stable, they will be public.

Reply:
Before they change, I will be pushing an update and won't deactivate these URLs until I see no traffic on them. :)

Reply:
Ok, got it, that makes sense.